examples/kepler-integration/src/components/export-video/export-video-panel-container.js

```diff
@@ -161,12 +157,26 @@ export class ExportVideoPanelContainer extends Component {
     });
     // setFileNameDeckAdapter(name.target.value);
   }
-  /* setQuality(resolution){
+  setQuality(resolution) {
+    // const {adapter, encoderSettings} = this.state
     this.setState({
       quality: resolution
     });
-    setResolution(resolution);
-  }*/
+    if (resolution === 'Good (540p)') {
+      this.setState({currentWidth: 960});
+      this.setState({currentHeight: 540});
+    } else if (resolution === 'High (720p)') {
+      this.setState({currentWidth: 1280});
+      this.setState({currentHeight: 720});
+    } else if (resolution === 'Highest (1080p)') {
+      this.setState({currentWidth: 1920});
+      this.setState({currentHeight: 1080});
```

Comment:
Call setState once. Three times in unnecessary and causes three rerenders instead of one.

Reply:
temp code. This still doesn't work so it'll be changed once I get things working